.github/workflows/build-test.yml

```diff
@@ -210,3 +214,33 @@ jobs:
                 CLOUD_PROVIDER: ${{ matrix.cloud }}
                 TARGET_DOCKER_TEST_IMAGE: ${{ matrix.image }}
               run: ./ci/test.sh
+
+    display_code_coverage:
+        needs: test-mac-required
+        name: Display code coverage
+        runs-on: macos-latest
+        strategy:
+          matrix:
+            cloud: [ 'AWS' ]
+        steps:
+          - uses: actions/checkout@v1
+          - uses: actions/setup-node@v1
+            with:
+              node-version: '12.x'
+          - uses: actions/setup-python@v1
+            with:
+              python-version: '3.7'
```

Comment:
do we need python here? I think this build step should be like the one described in lint.yml

Reply:
removed